From c1de15ad0a3154fa210e2bd43796d4b34194ea10 Mon Sep 17 00:00:00 2001
Message-Id: <c1de15ad0a3154fa210e2bd43796d4b34194ea10.1413837857.git.chang-joon.lee@intel.com>
In-Reply-To: <006497b7323eca32c2f489a1a9f662743c0d16d2.1413837855.git.chang-joon.lee@intel.com>
References: <006497b7323eca32c2f489a1a9f662743c0d16d2.1413837855.git.chang-joon.lee@intel.com>
From: Deepak S <deepak.s@intel.com>
Date: Tue, 22 Jul 2014 21:53:47 +0530
Subject: [PATCH 204/312] MUST_REBASE [VPG]: drm/i915: Enable DPST Interrupts
 for cherryview

!FOR_UPSTREAM: DPST feature is not to be upstreamed, as the corresponding
User mode component can not be open sourced.

Change-Id: I18a399e7493e210ef25b323118bf2bba152e08b3
Signed-off-by: Deepak S <deepak.s@intel.com>
---
 drivers/gpu/drm/i915/i915_irq.c |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/i915_irq.c b/drivers/gpu/drm/i915/i915_irq.c
index b258270..9db7673 100644
--- a/drivers/gpu/drm/i915/i915_irq.c
+++ b/drivers/gpu/drm/i915/i915_irq.c
@@ -3874,7 +3874,8 @@ static int cherryview_irq_postinstall(struct drm_device *dev)
 		I915_DISPLAY_PIPE_B_EVENT_INTERRUPT |
 		I915_DISPLAY_PIPE_C_EVENT_INTERRUPT;
 	u32 pipestat_enable = PLANE_FLIP_DONE_INT_STATUS_VLV |
-		PIPE_CRC_DONE_INTERRUPT_STATUS;
+		PIPE_CRC_DONE_INTERRUPT_STATUS |
+		PIPE_DPST_EVENT_STATUS;
 	unsigned long irqflags;
 	int pipe;
 
-- 
1.7.9.5

